### DISCUSSION TITLE
Still worth logging breaking changes from 3.1?

### DISCUSSION THREAD

**Discussion by tobiasbunyan** [dotnet/runtime#62777] [General] [answered]
I just uncovered a breaking change moving a project from 3.1 to 6.0 which does not seem to be covered in the [official docs](https://docs.microsoft.com/en-us/dotnet/core/compatibility/5.0) (did I miss it?).  Is there any point in still logging this or as it's now unsupported should I not bother?

Briefly, it relates to the default behaviour of Microsoft.Extensions.Configuration.UserSecrets.UserSecretsConfigurationExtensions.AddSecretsFile and the fact that in 3.1 it does not have an "optional" argument and supplies a forced "true" value whereas in 6.0 it now properly honours the default "false" optional argument from the parameterless invocation of AddUserSecrets<T>().  Thus this invocation in 3.1 does not fail if no secrets file is present (even if you have a secrets GUID in the project) but under 6.0 (and 5.0 presumably) it does.

**Comment by Joe4evr** [comment, marked as answer]
Duplicate of #61418
